[2025-05-02T06:36:04.892+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-02T06:36:04.936+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_to_gcs.upload_to_gcs manual__2025-05-02T06:35:40.647484+00:00 [queued]>
[2025-05-02T06:36:04.951+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_to_gcs.upload_to_gcs manual__2025-05-02T06:35:40.647484+00:00 [queued]>
[2025-05-02T06:36:04.953+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-05-02T06:36:05.013+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): upload_to_gcs> on 2025-05-02 06:35:40.647484+00:00
[2025-05-02T06:36:05.026+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=303) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-02T06:36:05.029+0000] {standard_task_runner.py:63} INFO - Started process 305 to run task
[2025-05-02T06:36:05.028+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_to_gcs', 'upload_to_gcs', 'manual__2025-05-02T06:35:40.647484+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/mysql_to_gcs_dag.py', '--cfg-path', '/tmp/tmp45uk31yd']
[2025-05-02T06:36:05.033+0000] {standard_task_runner.py:91} INFO - Job 28: Subtask upload_to_gcs
[2025-05-02T06:36:05.124+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_to_gcs.upload_to_gcs manual__2025-05-02T06:35:40.647484+00:00 [running]> on host c04d18144907
[2025-05-02T06:36:05.300+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_to_gcs' AIRFLOW_CTX_TASK_ID='upload_to_gcs' AIRFLOW_CTX_EXECUTION_DATE='2025-05-02T06:35:40.647484+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-02T06:35:40.647484+00:00'
[2025-05-02T06:36:05.303+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-02T06:36:05.313+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-05-02T06:36:05.322+0000] {base.py:84} INFO - Using connection ID 'google_cloud_default' for task execution.
[2025-05-02T06:36:07.760+0000] {gcs.py:560} INFO - File /tmp/data.csv uploaded to bronze_layer/data.csv in mysql-to-gcs-bronze-layer bucket
[2025-05-02T06:36:07.762+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-02T06:36:07.768+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-02T06:36:07.782+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=mysql_to_gcs, task_id=upload_to_gcs, run_id=manual__2025-05-02T06:35:40.647484+00:00, execution_date=20250502T063540, start_date=20250502T063604, end_date=20250502T063607
[2025-05-02T06:36:07.889+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-02T06:36:07.926+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-02T06:36:07.957+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
